csv-data-management/extract_data_from_csv.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import pandas as pd
+import numpy as np
+from os.path import join
+import json
+from connectors.core.connector import get_logger, ConnectorError
+from connectors.cyops_utilities.builtins import download_file_from_cyops
+from integrations.crudhub import make_request
+from .constants import LOGGER_NAME
+
+logger = get_logger(LOGGER_NAME)
+
+
+def extract_data_from_csv(config, params):
+    try:
+        endpoint = "files"
+        logger.info(params)
+        file_iri = handle_params(params)
+        file_path = join('/tmp', download_file_from_cyops(file_iri)['cyops_file_path'])
+        logger.info("file from code:" + file_path)
+        if params.get('columnNames'):
+            columnNames = params.get('columnNames')
+            columNames = columNames.split(",")
```

Comment:
spelling mistake? columNames Vs columnNames

Reply:
And we are not using columNames anywhere in if block

Reply:
typo corrected